portfolio/src/main/webapp/index.html

```diff
@@ -10,7 +10,7 @@
     type="text/javascript"></script>
   </head>
 
-  <body onload = "parseSomething()">
+  <body onload = "checkLoginStatus(); parseSomething();">
```

Comment:
There seem to be a bunch of changes in this PR that are not related to meeting scheduling. I would suggest moving the non-meeting related changes to a separate PR.

Reply:
The PR name and description are updated. :)

Reply:
The portfolio is a very different thing than the meeting scheduling so I think this is a case when we should move the changes to portfolio to a separate PR. We can do that together during the sync as an exercise for git. :-)